gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/PropertyMapStepTest.java

```diff
@@ -37,10 +38,12 @@
                 __.valueMap("name"),
                 __.valueMap("age"),
                 __.valueMap("name", "age"),
-                __.valueMap(true),
-                __.valueMap(true, "name"),
-                __.valueMap(true, "age"),
-                __.valueMap(true, "name", "age"),
```

Comment:
I don't think we should remove any old tests for the old signature. Those still need to be tested since we've only deprecated the approach.

Reply:
Can't mix those here, the hashCodes collide.